wftests/warrior_tests/testcases/framework_tests/execute_resume/one_invoked_execute_resume.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" ?>
+<Testcase>
+    <Details>
+        <Name>one_invoked_execute_resume</Name>
+        <Title>This testcase has one execute and resume onError</Title>
```

Comment:
the test case title says execute and resume onError .. what does this mean? 
instead it is better to say that "the test case has one execute and resume and the invoked step has executetype=invoked" as this is what is being done in the steps